### PR TITLE
Syncing with 78esr branch

### PR DESCRIPTION
**Pull request by msirringhaus**
* Adding SLE-12 special cases
* Adding s390x patches

### CODE REVIEW

File: firefox/MozillaFirefox.spec [hunk 1]
```diff
@@ -234,6 +256,8 @@ Development files for %{appname} to make packaging of addons easier.
 Summary:        Common translations for %{appname}
 Group:          System/Localization
 Provides:       locale(%{name}:ar;ca;cs;da;de;el;en_GB;es_AR;es_CL;es_ES;fi;fr;hu;it;ja;ko;nb_NO;nl;pl;pt_BR;pt_PT;ru;sv_SE;zh_CN;zh_TW)
+# This is there for updates from Firefox before the translations-package was split up into 2 packages
+Provides:       %{name}-translations
```
Comment (wrosenauer) [inline firefox/MozillaFirefox.spec:260]: I cannot even remember the time when this was the case. Is this really still relevant? Doesn't really hurt but ...

Comment (msirringhaus) [reply firefox/MozillaFirefox.spec:260]: This was removed some time ago and promptly caused a bug to pop up saying this is broken. So some customers seem to still upgrade from ancient versions.

File: firefox/MozillaFirefox.spec [hunk 2]
```diff
@@ -26,8 +33,10 @@
 %define branding       1
 %define devpkg         1
 
+%if 0%{?suse_version} > 1500
```
[the remark this answers is not in the document]
Comment (msirringhaus) [reply firefox/MozillaFirefox.spec:36]: I actually don't know. I thought, I removed the deactivation and got an unresolved dependency for xvfb-run which is only added if profiling is active, thus I assumed we do it everywhere.

File: firefox/MozillaFirefox.spec (hunk 1 above)
Comment (wrosenauer) [reply firefox/MozillaFirefox.spec:260]: ok

File: firefox/MozillaFirefox.spec (hunk 2 above)
[the remark this answers is not in the document]
Comment (wrosenauer) [reply firefox/MozillaFirefox.spec:36]: I will monitor this.

**Pull request merged**